Core/src/org/sleuthkit/autopsy/allcasessearch/AllCasesSearchDialog.java

```diff
@@ -155,26 +160,21 @@ protected void done() {
     private void initComponents() {
 
         correlationValueLabel = new javax.swing.JLabel();
-        correlationValueTextField = new javax.swing.JTextField();
         searchButton = new javax.swing.JButton();
         correlationTypeComboBox = new javax.swing.JComboBox<>();
         correlationTypeLabel = new javax.swing.JLabel();
         errorLabel = new javax.swing.JLabel();
         descriptionLabel = new javax.swing.JLabel();
         casesLabel = new javax.swing.JLabel();
+        jScrollPane1 = new javax.swing.JScrollPane();
```

Comment:
jScrollPane1? I would give the scroll pane a better name, you could also make the variable local instead of field.

Reply:
Changed the name.